auction-app-frontend/src/components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { search } from "src/assets/icons";
+
+import "./style.scss";
+
+const Searchbar = () => {
+  const [query, setQuery] = useState("");
+
+  const navigate = useNavigate();
+
+  const handleSearch = () => {
+    const url = new URL(window.location.href);
+    url.searchParams.set("search", query);
+
+    if (location.pathname.includes('/shop/')) {
```

Comment:
This can be done using ternary operator 
`location.pathname.includes('/shop/`) ? navigate(url.pathname + url.search) : navigate(`/shop/?search=${query}`);`

Always when you have those simple if/else statements, consider ternary operator, it can simplify your code.